### DISCUSSION TITLE
Welcome to reveal.js Discussions!

### DISCUSSION THREAD

**Discussion by hakimel** [hakimel/reveal.js#3060] [General]
## 👋 Welcome!
We're enabling Discussions to provide people with a forum to ask questions and get help from the reveal.js community. This space can be used to:
- 🙋 Ask questions about how to use reveal.js
- 🎙 Share a presentation you've made
- 💡 Suggest ideas for new features or changes

### Feature requests
I'm also planning to convert all 100+ existing feature request issues into discussions.

**Comment by DaveEveritt** [comment]
I’ve used Reveal (and markdown) for years, but am used to adding HTML tags within my markdown. I want to use an occasional `<small>` tag within larger blocks of markdown:

```html
<section data-markdown>
    # TITLE

    A regular paragraph

    - list item 1
    - list item 2
    - list item 3
 
    <small>
    [An external link](http://psychclassics.yorku.ca/Miller/)
    </small>
</section>
```

But the generated HTML removes the `small` tag, when I'd expect it to be nested inside a `p` tag - why/how/where is this happening? In other (non-Reveal) projects I’ve used Kramdown for Markdown conversion which allows this.

Alternatively, I'd like to add a class to a paragraph, say like this:

```markdown
{.myclass}
[An external link](http://psychclassics.yorku.ca/Miller/)  
```
